devtools-ci-index.yaml

```diff
@@ -159,11 +160,33 @@
 
 - job-template:
     name: '{ci_project}-{git_repo}-build-master'
-    <<: *job_template_build_master_defaults
+    scm:
+        - git:
+            url: https://github.com/{git_organization}/{git_repo}.git
```

Comment:
my idea is to keep the branch optional, but set it for those templates that use master branch. Another option is to keep a default value master